scripts/core/device.yml

```diff
@@ -365,7 +365,7 @@ etors:
     - name: SUB_GROUP_INDEPENDENT_FORWARD_PROGRESS
       desc: "[$x_bool_t] support sub group independent forward progress"
     - name: SUB_GROUP_SIZES_INTEL
-      desc: "[uint32_t[]] return an array of sub group sizes supported on Intel device"
+      desc: "[uint32_t[]] return an array of supported sub group sizes"
```

Comment:
When changing these files the `generate` build target needs to be invoked and the changes included in the pull request.